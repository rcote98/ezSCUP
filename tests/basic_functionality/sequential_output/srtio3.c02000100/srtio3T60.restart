2	2	2
5	3
Sr	Ti	O
0.00000000E+00	0.00000000E+00	0.00000000E+00	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	0	1	1	9.93525722E-03	2.48195477E-02	7.04785818E-03
0	0	0	2	2	1.05148586E-02	-6.62733752E-03	-6.17685999E-03
0	0	0	3	3	2.62272879E-01	2.63472350E-01	7.88230612E-03
0	0	0	4	3	1.18571516E-02	-1.32853808E-02	-2.76401017E-01
0	0	0	5	3	-3.83574783E-03	-2.47451247E-02	-2.75001553E-01
0	0	1	1	1	2.76294556E-03	-3.16999764E-02	9.27500306E-03
0	0	1	2	2	4.68652877E-03	-9.64884280E-03	-7.35974477E-03
0	0	1	3	3	-2.52107786E-01	-2.64213304E-01	3.63667696E-03
0	0	1	4	3	-5.44783706E-02	8.43537777E-03	2.88798481E-01
0	0	1	5	3	-2.81219199E-03	6.00300667E-02	2.78376523E-01
0	1	0	1	1	-2.63160086E-02	2.88603772E-03	-2.89917582E-02
0	1	0	2	2	-1.02560891E-02	-1.64032282E-02	2.50972676E-02
0	1	0	3	3	-3.29710580E-01	-2.72511048E-01	8.29838548E-03
0	1	0	4	3	-1.89321694E-02	-9.84863584E-03	2.97084631E-01
0	1	0	5	3	-1.71066268E-03	1.54570972E-02	3.30304321E-01
0	1	1	1	1	1.67794064E-02	-2.11677339E-02	-1.83547130E-03
0	1	1	2	2	-1.71015191E-02	-4.05203729E-03	9.63932159E-03
0	1	1	3	3	3.05558578E-01	2.77572891E-01	1.48685058E-02
0	1	1	4	3	3.44978351E-02	-2.14341265E-03	-2.66444229E-01
0	1	1	5	3	3.48366957E-03	-3.22855333E-02	-3.26906748E-01
1	0	0	1	1	4.35850170E-03	-1.66296967E-03	1.91208859E-02
1	0	0	2	2	9.59203869E-03	-2.58610618E-03	-4.58298136E-03
1	0	0	3	3	-2.18795601E-01	-3.41878314E-01	7.36026130E-03
1	0	0	4	3	-1.70183358E-02	-1.83130921E-03	3.23805853E-01
1	0	0	5	3	-6.73366291E-03	4.50290791E-02	2.54199798E-01
1	0	1	1	1	3.31205967E-03	-1.95749554E-03	2.28804388E-02
1	0	1	2	2	8.12895978E-04	1.05878138E-02	-4.68637283E-03
1	0	1	3	3	2.62190834E-01	3.15893221E-01	-2.15233711E-03
1	0	1	4	3	1.04812543E-02	-1.67904326E-03	-3.35291548E-01
1	0	1	5	3	1.31901113E-02	-2.56357567E-02	-2.77384619E-01
1	1	0	1	1	-8.42766143E-03	-4.35376056E-04	-3.68756387E-03
1	1	0	2	2	3.03300514E-03	-2.46554491E-03	-2.52291065E-02
1	1	0	3	3	3.19076018E-01	3.14583448E-01	-4.37832096E-03
1	1	0	4	3	1.88931799E-02	5.16514487E-03	-3.32781147E-01
1	1	0	5	3	-5.83273217E-04	-3.17631191E-02	-3.47933316E-01
1	1	1	1	1	6.24536508E-03	-2.62228936E-02	-1.55513623E-02
1	1	1	2	2	-7.65083850E-03	2.93162868E-02	-4.87293734E-03
1	1	1	3	3	-2.84104431E-01	-3.02159093E-01	-4.71367857E-03
1	1	1	4	3	-5.47996826E-02	7.36500000E-04	3.40451559E-01
1	1	1	5	3	1.84023891E-03	7.49237607E-02	3.04234590E-01
