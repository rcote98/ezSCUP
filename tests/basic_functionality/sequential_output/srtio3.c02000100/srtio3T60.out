
*****************************************************************
*                          Welcome to                           *
*                           SCALE-UP                            *
*                                                               *
*            Second-Principles Computational Approach           *
*                             for                               *
*                     Lattice and Electrons                     *
*                                                               *
*                       Version:         1.0.0.Gorbea           *
*                    Build date: 20200801T181                   *
*****************************************************************
*                    SCALE-UP is property of                    *
*              the University of Cantabria (Spain)              *
*    and the Luxembourg Institute of Science and Technology     *
*****************************************************************
* Authors:                                                      *
*                                                               *
*  - Pablo Garcia-Fernandez   garciapa@unican.es                *
*  - Jorge Iñiguez            jorge.iniguez@list.lu             *
*  - Javier Junquera          junqueraj@unican.es               *
*****************************************************************
* SCALE-UP is based on two earlier programs:                    *
*                                                               *
* -  SPLD ('Second-Principles Lattice Dynamics'), developed     *
*    at the ICMAB-CSIC (Spain) by J.C. Wojdel, M.P. Ljungberg   *
*    and J. Iñiguez, and distributed under MIT license.         *
*                                                               *
* -  MKII a second-principles electron code                     *
*    developed at the University of Cantabria by                *
*    P. Garcia-Fernandez  and J. Junquera.                      *
*****************************************************************
*                                                               *
* Bibliography:                                                 *
*                                                               *
*  P. Garcia-Fernandez et al. Phys. Rev. B 93, 195137 (2016)    *
*  J. C. Wojdel et al., J. Phys. Cond. Matt. 25, 305401 (2013)  *
*****************************************************************
Reading parameter file... lattice... done!
Reading restart file     srtio3T60.restart                                                               

Basic data:
  Number of CPUs:               1
  Run mode:            Montecarlo
  Lattice:                 Active
  Electrons:           NOT Active
  External electric field, static. Field components (V/m):   0.000000E+00   0.000000E+00   0.000000E+00

Reference system:
 Supercell data:
  Lattice constants (Ang):   7.690   7.690   7.690
  Lattice angles (degree):  90.000  90.000  90.000
 Supercell size:   2   2   2

 Total number of atoms          :      40
 Total number of orbitals       :       0
 Total number of active orbitals:       0

 Atom in unit cell:
 Atom   Label       Mass  Charge 
    1   Sr         87.60     0.0
    2   Ti         47.90     0.0
    3   O          16.00     0.0
    4   O          16.00     0.0
    5   O          16.00     0.0

 Orbitals in unit cell:
 Orbital   Label  

 Background dielectric tensor:
    6.2319     0.0000     0.0000
    0.0000     6.2319    -0.0000
    0.0000    -0.0000     6.2319

Initial values (basic output MC simulation)
Energy (eV):            -0.298487E+05
Polarization (C/m2):    -0.000554   -0.004130   -0.004541
Strain:                  0.000000    0.000000    0.000000    0.000000    0.000000    0.000000

LT:    Iter       Temp(K)    Energy(eV)   Pol_x(C/m2)   Pol_y(C/m2)   Pol_z(C/m2) Av_Pl_x(C/m2) Av_Pl_y(C/m2) Av_Pl_z(C/m2)       Strn_xx       Strn_yy       Strn_zz       Strn_yz       Strn_xz       Strn_xy    Av_Strn_xx    Av_Strn_yy    Av_Strn_zz    Av_Strn_yz    Av_Strn_xz    Av_Strn_xy 
LT:      10  0.600000E+02 -0.298485E+05  0.100242E-01  0.334248E-01  0.270485E-01 -0.679118E-02  0.518696E-02  0.173285E-01  0.000000E+00  0.000000E+00 -0.211896E-02  0.368504E-03 -0.176054E-02  0.000000E+00  0.000000E+00  0.000000E+00 -0.119360E-02  0.112689E-02  0.131715E-03  0.000000E+00 
LT:      20  0.600000E+02 -0.298485E+05 -0.801205E-02  0.503549E-01  0.533254E-01 -0.417704E-02  0.192100E-01  0.278061E-01  0.000000E+00  0.000000E+00 -0.387091E-02  0.157940E-02  0.340350E-03  0.000000E+00  0.000000E+00  0.000000E+00 -0.237621E-02  0.952682E-03 -0.483744E-03  0.000000E+00 
LT:      30  0.600000E+02 -0.298485E+05  0.415182E-01  0.374803E-01  0.226038E-01  0.373661E-02  0.278404E-01  0.273881E-01  0.000000E+00  0.000000E+00 -0.359669E-02  0.234302E-02 -0.413650E-02  0.000000E+00  0.000000E+00  0.000000E+00 -0.292011E-02  0.978473E-03 -0.110738E-02  0.000000E+00 
LT:      40  0.600000E+02 -0.298485E+05  0.386767E-02  0.430568E-02  0.378746E-02  0.850611E-02  0.228600E-01  0.264788E-01  0.000000E+00  0.000000E+00 -0.499532E-02  0.261630E-02 -0.225727E-02  0.000000E+00  0.000000E+00  0.000000E+00 -0.287681E-02  0.159317E-02 -0.157900E-02  0.000000E+00 
LT:      50  0.600000E+02 -0.298484E+05 -0.262570E-02  0.151910E-01 -0.289501E-01  0.620690E-02  0.225629E-01  0.205740E-01  0.000000E+00  0.000000E+00 -0.501703E-02  0.462311E-02 -0.156729E-02  0.000000E+00  0.000000E+00  0.000000E+00 -0.316153E-02  0.185632E-02 -0.168904E-02  0.000000E+00 
LT:      60  0.600000E+02 -0.298485E+05  0.155642E-01  0.287798E-01 -0.163574E-01  0.355407E-02  0.232672E-01  0.130857E-01  0.000000E+00  0.000000E+00 -0.307206E-02  0.443232E-02  0.286145E-03  0.000000E+00  0.000000E+00  0.000000E+00 -0.348182E-02  0.223901E-02 -0.147097E-02  0.000000E+00 
LT:      70  0.600000E+02 -0.298485E+05 -0.399312E-01 -0.769750E-02 -0.452651E-02  0.408350E-03  0.181557E-01  0.104370E-01  0.000000E+00  0.000000E+00 -0.380554E-02  0.505620E-02  0.995095E-03  0.000000E+00  0.000000E+00  0.000000E+00 -0.332802E-02  0.266575E-02 -0.107713E-02  0.000000E+00 
LT:      80  0.600000E+02 -0.298485E+05  0.248353E-03  0.193660E-01 -0.990363E-02  0.129276E-02  0.182282E-01  0.975156E-02  0.000000E+00  0.000000E+00 -0.551865E-02  0.224302E-02  0.498867E-02  0.000000E+00  0.000000E+00  0.000000E+00 -0.344890E-02  0.282715E-02 -0.515460E-03  0.000000E+00 
LT:      90  0.600000E+02 -0.298484E+05 -0.113837E-01 -0.388839E-02  0.135253E-01 -0.159353E-02  0.162468E-01  0.939986E-02  0.000000E+00  0.000000E+00 -0.559478E-02  0.274799E-03  0.791324E-02  0.000000E+00  0.000000E+00  0.000000E+00 -0.384573E-02  0.260288E-02  0.296310E-03  0.000000E+00 
LT:     100  0.600000E+02 -0.298484E+05 -0.321079E-02  0.150329E-01  0.255903E-01 -0.132594E-02  0.150291E-01  0.964510E-02  0.000000E+00  0.000000E+00 -0.355296E-02 -0.755481E-02  0.582408E-02  0.000000E+00  0.000000E+00  0.000000E+00 -0.361914E-02  0.196773E-02  0.914758E-03  0.000000E+00 

Final values (basic output MC simulation)
Energy (eV):            -0.298484E+05
Polarization (C/m2):    -0.003211    0.015033    0.025590
Strain:                  0.000000    0.000000   -0.003553   -0.007555    0.005824    0.000000

-----------------------------------------------------------------
 Backing up the input read by fdf:

System_name     srtio3T60
Print_routineinfo         5     # default value
Print_routinedepth         0     # default value
Print_inout         0     # default value
Print_interactions         0     # default value
Print_k-vectors         0     # default value
Print_energies         1     # default value
Print_wavefunctions        -1     # default value
Print_electrostatic         1     # default value
Print_orbocc         1     # default value
Print_bands        -1     # default value
Print_forces        -1     # default value
Print_detail        -1     # default value
Print_Just_Geometry        -1     # default value
Print_Velocity         0     # default value
Print_Space_Hamiltonian        -1     # default value
Print_space_denmat        -1     # default value
Print_Geometry         1     # default value
Print_geoformat     normal     # default value
Print_test_file        -1     # default value
Print_local_lattice_energy        -1     # default value
Print_local_current        -1     # default value
Print_std_electron_nsteps         1     # default value
Print_std_lattice_nsteps             10
Print_std_energy              T
Print_std_polarization              T
Print_std_current              F     # default value
Print_std_strain              T
Print_std_av_polarization              T
Print_std_av_strain              T
Print_std_av_energy              F     # default value
Print_std_delta_energy              F     # default value
Print_std_temperature              T     # default value
Print_std_maximum_force              F     # default value
Print_std_maximum_stress              F     # default value
Print_std_mdd              T     # default value
Print_std_fermi              T     # default value
Print_std_pulay              T     # default value
Print_std_neh              T     # default value
Print_std_el_pol              T     # default value
Print_std_micro              F     # default value
run_mode     monte_carlo
Print_std_mdd              F     # default value
Print_std_fermi              F     # default value
Print_std_pulay              F     # default value
Print_std_neh              F     # default value
Print_std_electron_nsteps     10000     # default value
No_electron
%block fix_strain_component
 T T F F F T
T                                                                                             T
T                                                                                             T
F                                                                                             F
F                                                                                             F
F                                                                                             F
T                                                                                             T
Parameter_file     srtio3_full_lat.xml
%block Supercell
 2 2 2
%endblock Supercell
electric_field_scale          1.000000000     v/m     # default value
%block static_electric_field
 0.0 0.0 0.0
temperature         0.1000000000E-98 kelvin     # default value
%block external_stress
 0.0 0.0 0.0 0.0 0.0 0.0
Charge_tolerance         0.1000000000E-07     # default value
Geometry_restart
Geometry_restart     srtio3T60.restart
MC_nsweeps
MC_nsweeps            100
MC_temperature
MC_temperature          60.00000000         kelvin
# above item originally: MC_temperature          60.00000000     kelvin
MC_annealing_rate          1.000000000
MC_init_step_d         0.9448629925E-01 bohr     # default value
MC_init_step_s         0.1889725985E-02 bohr     # default value
MC_max_step_d         0.9448634389           bohr
# above item originally: MC_max_step_d         0.5000000000     ang
MC_min_step_d         0.1889725985E-05 bohr     # default value
MC_target_acc_d         0.5000000000         # default value
MC_target_acc_s         0.5000000000         # default value
MC_strains              T
n_write_mc
n_write_mc             10
Print_std_lattice_nsteps
*****************************************************************
 Memory usage:
  Function             Total_memory(Mb)  Max_memory(Mb)      Node
  setup_main                     0.0256          0.0256         0
  setup_basicvariables           0.0056          0.0056         0
  setup_latt_hamil               0.0014          0.0014         0
  setup_init_data_node           0.0256          0.0256         0
  hel_setup_structfactors        0.0256          0.0256         0
  hlt_create_supercell           0.0014          0.0014         0

 Timing statistics (seconds)
  Function                       Wall   Max_Wall    Max_CPU  Node
  setup_main                     3.76       3.76       3.72     0
  setup_readinput                3.54       3.54       3.51     0
  montecarlo_main                0.22       0.22       0.21     0
  montecarlo_sweep               0.21       0.21       0.21     0
  io_readparameterfile           3.54       3.54       3.51     0
  setup_latt_hamil               0.22       0.22       0.22     0
  hlt_add_harmonic_didi          0.22       0.22       0.22     0

 Total memory usage:     0.03 MB
 Total WALL:           3.974 seconds
*****************************************************************
*                                                               *
*               Succesfully ending SCALE-UP                     *
*                                                               *
*****************************************************************
